python/pfs/pipe2d/generateCommands.py

```diff
@@ -478,6 +499,23 @@ def clean(self, fout: TextIO, dataDir: str, rerun: str):
         command = ["rm", "-r", "-f", os.path.join(dataDir, "rerun", rerun)]
         print(shellCommand(command), file=fout)
 
+    @property
+    def outputCalibType(self) -> str:
```

Comment:
Looking below, I see that this isn't really about the calib type, but the directory the calibs end up under, so I wonder if you should rename it to `outputCalibSubDir` or similar?

Reply:
I renamed it to `outputSubdir`.